.github/workflows/phpunit.yml

```diff
@@ -28,7 +28,7 @@ jobs:
                   php-version: "${{ matrix.php-version }}"
 
             - name: "Cache dependencies installed with composer"
-              uses: "actions/cache@v2"
+              uses: "actions/cache@v4"
```

Comment:
Error: This request has been automatically failed because it uses a deprecated version of `actions/cache: v2`. Please update your workflow to use v3/v4 of actions/cache to avoid interruptions. Learn more: https://github.blog/changelog/2024-12-05-notice-of-upcoming-releases-and-breaking-changes-for-github-actions/#actions-cache-v1-v2-and-actions-toolkit-cache-package-closing-down